apps/src/sites/studio/pages/blocks/index.js

```diff
@@ -9,9 +9,43 @@ import {shrinkBlockSpaceContainer} from '@cdo/apps/templates/instructions/utils'
 import animationList, {
   setInitialAnimationList
 } from '@cdo/apps/p5lab/redux/animationList';
-import defaultSprites from '@cdo/apps/p5lab/spritelab/defaultSprites.json';
 import {getStore, registerReducers} from '@cdo/apps/redux';
 
+// Blocks in the index are readonly. Use a single costume and a single background for previews.
+let defaultThumbnails = {
```

Comment:
i'm concerned about having animation metadata contained in this file because it creates an implicit dependency on existing animations. 

there's probably many ways to handle this, but one thought: could we create a new file that makes this dependency a little more explicit? it would be similar to the existing `defaultSprites` file and would remove the need to define animations inline here. (i think the file could also be JS rather than JSON, which would give us more flexibility to add comments around why the file is needed and where it's used.)